### PR TITLE
Tpwdcee 7030 configuration

### PR DESCRIPTION
**Pull request by brkicadis**

### CODE REVIEW

File: tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php [hunk 1]
```diff
@@ -227,4 +263,53 @@ public function seeInFieldBasedOnElementName($elName, $elValue, $elLocator, $pag
...
+        $mappedPaymentMethod = $this->mapPaymentMethodTransactionField($paymentMethod);
+
+        $this->seeInDatabase(
+            static::TRANSACTION_TABLE_NAME,
+            [static::TRANSACTION_ORDER_ID => $this->getTransactionFieldsFromSuccessPage()['order_id'],
+            static::TRANSACTION_CURRENCY => $this->getTransactionFieldsFromSuccessPage()['currency'],
+            static::TRANSACTION_TYPE_COLUMN_NAME=> $paymentAction,
+            static::TRANSACTION_AMOUNT => $this->getTransactionFieldsFromSuccessPage()['amount'],
+            static::TRANSACTION_PAYMENT_METHOD=> strtolower($mappedPaymentMethod),
+            static::PARENT_TRANSACTION_ID => '',
```
Comment (tomazpu) [inline tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php:282]: In the US point 4 its written to check that the parent transaction id is not empty

File: tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php [hunk 2]
```diff
@@ -227,4 +263,53 @@ public function seeInFieldBasedOnElementName($elName, $elValue, $elLocator, $pag
...
+        return [
+            'order_id' => $orderId,
+            'currency' => str_replace($currency, $this->currencyFields[$currency], $currency),
+            'amount' => str_replace($currency, "", $amount)
+        ];
+    }
+
+    /**
+     * @param $paymentMethod
+     * @return mixed
```
Comment (tomazpu) [inline tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php:305]: is not always a string returned?

File: tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php [hunk 3]
```diff
@@ -227,4 +263,53 @@ public function seeInFieldBasedOnElementName($elName, $elValue, $elLocator, $pag
...
+            static::TRANSACTION_PAYMENT_METHOD=> strtolower($mappedPaymentMethod),
+            static::PARENT_TRANSACTION_ID => '',
+            static::TRANSACTION_STATE . ' !=' => '',
+            static::TRANSACTION_ID . ' !=' => '',
+            static::TX_ID . ' !=' => ''
+            ]
+        );
+    }
+
+    public function getTransactionFieldsFromSuccessPage()
```
Comment (tomazpu) [inline tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php:290]: the method has no annotation with a return type or anything.

File: tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php (hunk 1 above)
Comment (brkicadis) [reply tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php:282]: 4. Parent transaction ID: per default it is empty

File: tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php (hunk 2 above)
Comment (brkicadis) [reply tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php:305]: it is a string returned, fixed.

File: tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php (hunk 3 above)
Comment (brkicadis) [reply tests/_support/Step/Acceptance/ShopSystem/WoocommerceBackendStep.php:290]: fixed.

**Pull request merged**